fontbakery-check-description.py

```diff
@@ -20,7 +20,7 @@
 import os
 from TargetFont import TargetFont
 from FontBakeryCheckLogger import FontBakeryCheckLogger
-import fontbakery_checks as checks
+import FontBakeryChecks as checks
```

Comment:
ok, cool! I was indeed unsure this was really a good choice of naming style. Thanks for pointing that out. I was a bit unhappy with the name fontbakery_checks.py because it was too similar to the name scheme we use for the command line tools such as fontebakery-check-tty.py

I think I'll rename this module to fb_checks.py then. Or even simply checks.py